src/thunderbots/software/ai/world/robot.h

```diff
@@ -178,6 +181,41 @@ class Robot
     AngularVelocity estimateAngularVelocityAtFutureTime(
         const Duration &duration_in_future) const;
 
+    /**
+     * Gets the buffer which holds all the previous position states of the robot
```

Comment:
For all these new "getPrevious..." functions they should include comments indicating what order the previous data is returned in. ie. is it in order from the newest to oldest data, or oldest to newest?

Reply:
done